scss/_patterns_segmented-control.scss

```diff
@@ -0,0 +1,60 @@
+@mixin vf-p-segmented-control {
+  %vf-button-group-borders {
+    border-bottom: 0;
```

Comment:
A bit worried that we're moving the borders like this. Why not keep them but make them transparent if needed?

Reply:
I'm also bit worried about taking the border out of the buttons. It seems to be affecting focus a bit (buttons are smaller) and overall will make it harder to keep this component aligned with buttons.

It seems that maybe we need to refactor buttons styles a bit to make it easier to reuse their styling in other places.